### PR TITLE
Add tile layer type and format to geodata schema

### PR DESCRIPTION
**Pull request by srezacova**
## Description

- geodata schema updated so we can use "tiles" in format and "tile" in geometryDataTypes in GeoData config. Also added notice to changelog, that tileDataUrl and tileAttributions are being deprecated and changed FE according to this. 

## Motivation and Context

- after mongoDB refactor to Postgresql, the filter in layers is changed(for postgre version) so now it expect all layers to have georeferenceId Point and Tile layers included, so geoData are needed for these layers now and we need format and layer type for Tile layer
- this change also affected other part of code, so now we can use 'apiUrl' key instead of 'tileDataUrl' key and 'metadata.description' key instead of 'tileAttributions' key so tileDataUrl and tileAttributions are being deprecated
- after merging this branch, create a new release if possible

##  How to test
- add metadata attributions prop to MapLayers and GeoData config, run app and initial load and see if you can find them in map info.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have checked to ensure that there aren't other open [Pull Requests](../../../pulls) for the same update/change.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: frontend/src/ol/staticLayers/layerTypes/tiles.js [hunk 1]
```diff
@@ -9,8 +9,10 @@ const tileLayer = (layerData) => {
     featureId: layerData.featureId,
     type: layerData.layerType,
     source: new XYZ({
-      url: layerData.tileDataUrl,
-      attributions: layerData.tileAttributions,
+      url: layerData.tileDataUrl || layerData.geoDataUrl,
+      attributions:
+        layerData.tileAttributions ||
+        (layerData.metadata && layerData.metadata.geoMetadata && layerData.metadata.geoMetadata.description),
```
Comment (bariela) [inline frontend/src/ol/staticLayers/layerTypes/tiles.js:15]: should be attribution, not description, we should keep description for human-friendly explanation :)

**Review by bariela — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - new color style type 'intervals' [117](https://github.com/openkfw/Oscar/pull/120)
+- 'metadata.attributions' key in ./initial-data-load/data/{country}/GeoData.yml configuration file as a substite for 'tileAttributions' key on the top level that is now deprecated. You can find out more about these settings in [docs](./doc/data-structures/config-files.md#geoData.yml). [#146](https://github.com/openkfw/Oscar/pull/146/files)
```
[the remark this answers is not in the document]
Comment (srezacova) [reply CHANGELOG.md]: yeah good catch, I changed it.

File: CHANGELOG.md [hunk 3]
```diff
@@ -43,6 +45,7 @@ Introducing option to use postgresql as database, with the exception of function
 
 - route '/api/staticLayers', use '/api/dataLayers' [#111](https://github.com/openkfw/Oscar/issues/111)
 - 'COUNTRY' environment variable for initial-data-load, use 'DATASET' instead
+- 'tileDataUrl' and 'tileAttributions' keys on the top level in ./initial-data-load/data/{country}/GeoData.yml configuration file. Use 'apiUrl' key instead of 'tileDataUrl' key and 'metadata.attributions' key instead of 'tileAttributions' key. You can find out more about these settings in [docs](./doc/data-structures/config-files.md#geoData.yml). [#146](https://github.com/openkfw/Oscar/pull/146/files)
```
Comment (bariela) [inline CHANGELOG.md:48]: this was not part of the release, looks like incorrect merge

Comment (srezacova) [reply CHANGELOG.md]: fixed

**Review by bariela — APPROVED** [review]

**Pull request merged**